controller/apicontroller.php

```diff
@@ -107,6 +107,11 @@ public function collection() {
 		$artists = array();
 		foreach ($allTracks as $track) {
 			$artist = &$allArtistsById[$track->getArtistId()];
+			$albumArtists = $this->albumBusinessLayer->find($track->getAlbumId(), $this->userId);
+
+			if($albumArtists->getArtistIds()[0]!=$artist['id']) {
+				$artist = &$allArtistsById[$albumArtists->getArtistIds()[0]];
+			}
```

Comment:
Again, Album Artist is a separate tag altogether and its value should not be taken from the Artist Tag. This would result in a wrong value for an Album containing collection of songs of various artists. Please refer to #503 for the changes